articles/protocols/saml/saml-configuration/special-configuration-scenarios/idp-initiated-sso.md

```diff
@@ -39,22 +35,20 @@ For example, an organization might set up a portal to ensure that users navigate
 3. If successfully authenticated, the user clicks on the appropriate link;
 4. The user is redirected to the service provider with a SAML assertion.
 
-## Auth0 as Service Provider Where IdP Initiates SSO
+## Risks of Using an IdP-Initiated SSO Flow
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/hZGYWeBvZQ8" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+In general, it is recommended to use Service-Provided flows whenever available. When an application actively requests an authentication as a first step (as is the case for SP-Initiated flows), it can check that the authentication response matches the original request. 
+In IdP-initiated scenarios, on the other hand, the application can not verify that the user actually started the flow. Because of this, the IdP-Initiated flow opens the possibility of an [Login CSRF attack](https://support.detectify.com/customer/portal/articles/1969819-login-csrf), where an attacker can trick a legitimate user into using a session created by the attacker. 
 
-If Auth0 acts as the service provider, you need to make the following changes to support IdP-initiated SSO:
+Login CSRF attacks are generally less of a concern in enterprise-only scenarios, as any would-be attacker would have to come from the same directory of users. But it's definitely a security concern in consumer-facing applications, so we strongly advise against enabling IdP-Inititated flows on SAML connections.
```

Comment:
Is this block necessary? I'm wondering if this'll give readers a false sense of security.

Reply:
Good point. Just changed the message to be more clear.